cmd/exocored/root.go

```diff
@@ -132,7 +143,24 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		a.appExport,
 		addModuleInitFlags,
 	)
-
+	startCmd, _, _ := rootCmd.Find([]string{cmdStartName})
+	startCmd.Flags().Bool(flagOracle, false, "enable oracle feeder")
+	startCmd.Flags().String(flagMnemonic, "", "set validator consensus key's mnemonic")
+	preRunE := startCmd.PreRunE
+	// add preRun to run price-feeder first before starting the node
+	startCmd.PreRunE = func(cmd *cobra.Command, args []string) error {
+		if enableOracle, _ := cmd.Flags().GetBool(flagOracle); enableOracle {
+			mnemonic, _ := cmd.Flags().GetString(flagMnemonic)
+			clientCtx := cmd.Context().Value(client.ClientContextKey).(*client.Context)
+			cmdFeeder := exec.Command(path.Join(clientCtx.HomeDir, feederBianry), flagFeederConfig, path.Join(clientCtx.HomeDir, confPath, confOracle), flagFeederSource, path.Join(clientCtx.HomeDir, confPath), flagFeederMnemonic, mnemonic, cmdStartName)
+			cmdFeeder.Stdout = os.Stdout
+			cmdFeeder.Stderr = os.Stderr
+			if err := cmdFeeder.Start(); err != nil {
```

Comment:
Concurrency scenario: 
price-feeder is started asynchronously, the code does not wait for it to initialize before continuing execution. 
Will this cause the main program to start working before price-feeder is ready ?

Reply:
Yes it would be possible, but it's fine for price-feeder be ready later than exocored